yarn.lock

```diff
@@ -0,0 +1,11896 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
please delete this file and add it to `.gitignore`

Reply:
ok i did it